sdk/openai/Azure.AI.OpenAI/README.md

```diff
@@ -106,41 +148,43 @@ foreach (var prompt in examplePrompts)
 }
 ```
 
-### Generate Chatbot Responses With Token
+### Summarize Text with Completion
 
-The `GenerateChatbotResponsesWithToken` method authenticates using a DefaultAzureCredential, then generates text responses to input prompts.
+The `SummarizeText` method generates a summarization of the given input prompt.
 
-```C# Snippet:GenerateChatbotResponsesWithToken
+```C# Snippet:SummarizeText
 string endpoint = "http://myaccount.openai.azure.com/";
+string textToSummarize = @"
+    Two independent experiments reported their results this morning at CERN, Europe's high-energy physics laboratory near Geneva in Switzerland. Both show convincing evidence of a new boson particle weighing around 125 gigaelectronvolts, which so far fits predictions of the Higgs previously made by theoretical physicists.
+
+    ""As a layman I would say: 'I think we have it'. Would you agree?"" Rolf-Dieter Heuer, CERN's director-general, asked the packed auditorium. The physicists assembled there burst into applause.
+:";
 OpenAIClient client = new OpenAIClient(new Uri(endpoint), new DefaultAzureCredential());
 
-List<string> examplePrompts = new(){
-    "How are you today?",
-    "What is Azure OpenAI?",
-    "Why do children love dinosaurs?",
-    "Generate a proof of Euler's identity",
-    "Describe in single words only the good things that come into your mind about your mother.",
-};
+string summarizationPrompt = @$"
+    Summarize the following text.
 
-foreach (var prompt in examplePrompts)
-{
-    Console.Write($"Input: {prompt}");
-    var request = new CompletionsOptions();
-    request.Prompt.Add(prompt);
+    Text:
+    """"""
+    {textToSummarize}
+    """"""
 
-    Completions completion = client.GetCompletions("myModelDeployment", request);
-    var response = completion.Choices[0].Text;
-    Console.WriteLine($"Chatbot: {response}");
-}
-```
+    Summary:
+";
 
-## Troubleshooting
+Console.Write($"Input: {summarizationPrompt}");
+var request = new CompletionsOptions();
+request.Prompt.Add(summarizationPrompt);
 
-Describe common errors and exceptions, how to "unpack" them if necessary, and include guidance for graceful handling and recovery.
+Completion completion = client.GetCompletions("myModelDeployment", request);
```

Comment:
use `Response<Completions>` also I don't think this would compile as is make sure its backed by a cs file correctly

Reply:
This does in fact compile, see the "[Ignore("Only verifying that the sample builds")]" header in the sample file (and the library code for Response for why it works), but we can use the "completionResponse.Value.Choices[0].Text;" syntax if you feel that's more helpful. Your call.

Reply:
The issue with using `Completion` is that people copy and paste samples and sometimes they change to `var`.

If they ended up with `var completion = client.GetCompletions("myModelDeployment", request);` the next line `string response = completion.Choices[0].Text;` will not compile since the type of the completion variable is now a `Response<Completions>`.  The line as written is taking advantage of an implict conversion of `Response<Completions>` -> `Completions` which is handy but isn't very discoverable and can lead to some frustration when copy paste isn't working.

Reply:
Thanks for the explanation, Michael. :-)